src/OrchardCore.Modules/OrchardCore.Alias/Migrations.cs

```diff
@@ -33,11 +32,11 @@ public int Create()
             );
 
             SchemaBuilder.AlterIndexTable<AliasPartIndex>(table => table
-                .CreateIndex("IDX_AliasPartIndex_Alias", "Alias", "Published", "Latest")
+                .CreateIndex("IDX_AliasPartIndex_DocumentId", "DocumentId", "Alias", "ContentItemId", "Latest", "Published")
```

Comment:
For infos @jtkech this goes over 768. (which is why I never put the `ContentItemId` in the index originally.

I ran the functional tests against this branch. Here's the link https://github.com/OrchardCMS/OrchardCore/actions/runs/468293972 to the result. (do you know how to run them manually? Just go to actions, pick the functional tests, and choose your branch).

They're not perfect, as they only use a recipe, so things like `OpenId` don't get started, but still useful.

Locally, I'm just using a docker of mysql and postgres, and tweak some of the integration tests to initialize features, when necesary. We still need to get a better way of testing features. But extending the functional tests, to enable them, seems to be the preferred way.